org.sofproject.gst.topo/src/org/sofproject/gst/topo/ui/preferences/ConnectionPreferences.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2019, Intel Corporation
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *   * Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *   * Redistributions in binary form must reproduce the above copyright
+ *     notice, this list of conditions and the following disclaimer in the
+ *     documentation and/or other materials provided with the distribution.
+ *   * Neither the name of the Intel Corporation nor the
+ *     names of its contributors may be used to endorse or promote products
+ *     derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+package org.sofproject.gst.topo.ui.preferences;
+
+import org.eclipse.core.runtime.preferences.InstanceScope;
+import org.eclipse.jface.preference.FieldEditorPreferencePage;
+import org.eclipse.jface.preference.IPreferenceStore;
+import org.eclipse.jface.preference.StringFieldEditor;
+import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.IWorkbenchPreferencePage;
+import org.eclipse.ui.preferences.ScopedPreferenceStore;
+
+public class ConnectionPreferences extends FieldEditorPreferencePage implements IWorkbenchPreferencePage {
+
+	StringFieldEditor ipFieldEditor;
+	StringFieldEditor userFieldEditor;
+
+	@Override
+	public void init(IWorkbench workbench) {
+		IPreferenceStore store = new ScopedPreferenceStore(InstanceScope.INSTANCE,
+				"org.sofproject.gst.topo.ui.preferences");
+		setPreferenceStore(store);
+
+	}
+
+	@Override
+	protected void createFieldEditors() {
+		ipFieldEditor = new StringFieldEditor(ConnectionPreferencesConst.CONN_ADDR, "Address for all projects: ",
+				getFieldEditorParent());
+		addField(ipFieldEditor);
+
+		userFieldEditor = new StringFieldEditor(ConnectionPreferencesConst.CONN_USERNAME, "Username for all projects: ",
```

Comment:
"Last login name for Audio Dev projects: "?

Reply:
We don't want use 'audio'. I will change it to Last login name for Dev projects.